devtools/common/src/main/filtered/extensions.json

```diff
@@ -105,6 +107,7 @@
   },
   {
     "name": "Hibernate Validator",
+    "shortName": "bean-validation",
```

Comment:
I don't think people write bean-validation. It's `Bean Validation`

Reply:
Fixed